### DISCUSSION TITLE
mbtiles breakdown

### DISCUSSION THREAD

**Discussion by BenOnTrack** [systemed/tilemaker#533] [Q&A] [answered]
Hi, 

I'm wondering what's best between : 
- breaking down an area (let's say a country that you'd like to render) into multiples mbtiles (for instance, a basemap with landuse, landcover, water, ... and then one for the poi, one for transport, ...)
- a single mbtiles containing all foreseen layers 

when it comes to size / rendering ?

In other words, it it similar, in terms of performance, to render a single mbtiles filtered on specific layers versus multiple mbtiles containing all those layers. 

I understand this question might be asked to the server/render community (tileserver, marvin, ...) but since it's impacting my process of tiles generation with tilemaker I'd be happy to have your thoughts. 

I've seen that people used to actually merge mbtiles (instead of keeping them apart) so I'm curious if that means it's better to actually keep a single mbtiles in the end.

Thanks in advance for your support!

**Comment by systemed** [comment, marked as answer]
There's no hard and fast rule on this really. I tend to have one main .mbtiles containing my basemap layers, and then create another one if there are any overlays I want to put on this map.

I'm not aware of any significant rendering performance impact from having multiple sources. Obviously there'll be some extra HTTP fetch time if you're loading two tiles for each location rather than just one.

**Reply by BenOnTrack** [reply]
Thanks for your feedback!
